tensorboard/components/tf_markdown_view/tf-markdown-view.ts

```diff
@@ -46,6 +46,9 @@ class TfMarkdownView extends LegacyElementMixin(PolymerElement) {
       #markdown > p:last-child {
         margin-bottom: 0.3em;
       }
+      #markdown p {
+        white-space: pre;
```

Comment:
This will have the unintended consequence of meaning that actual paragraphs of text will not be wrapped in a sane manner.

For the use case you are investigating, I think we should understand why the markdown renderer is wrapping some text with `<pre><code></code></pre>` and other code with just `<p>`.

I suspect it has to do with the non alpha-numeric text included in the results but that is just speculation.

Markdown Rendering is here:
https://github.com/tensorflow/tensorboard/blob/master/tensorboard/plugin_util.py

Reply:
Thanks Brian, that's a good point. I'll look into that thing from the markdown converter until, but also, how about using `white-space: break-spaces;` instead? Looks like this pretty much solves the problem by allowing text to wrap, but preserving spaces: https://developer.mozilla.org/en-US/docs/Web/CSS/white-space

Reply:
That result with the <pre> tags seems to come from the markdown library. I don't think it's worth trying to figure out how that works. I'm more inclined to just go with the `white-space: break-spaces;` solution. If users have an issue with this, we can add a checkbox control later, similar to the "markdown" one, tho I imagine most users would want to see the spaces if they're there.